### ISSUE TITLE
Test flake - KafkaNativeContainerIT.testKraftClusterBothControllers sometimes fails 

### ISSUE THREAD

**Issue by k-wall** [ozangunalp/kafka-native#6]
CI failed with this error.   The error suggests that the brokers weren't able to find one and other.
I haven't been able to reproduce it so far.
 
```
but was: 1 within 30 seconds.
	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)
	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)
	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:985)
	at org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:769)
	at com.ozangunalp.kafka.test.container.KafkaNativeContainerIT.verifyClusterMembers(KafkaNativeContainerIT.java:65)
	at com.ozangunalp.kafka.test.container.KafkaNativeContainerIT.testKraftClusterBothControllers(KafkaNativeContainerIT.java:183)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
```

**Comment by k-wall** [comment]
We really need the test framework to gather the container logs to be able to diagnose what went wrong.

**Comment by k-wall** [comment]
Should be fixed by 2nd commit on #7
